test/CMakeLists.txt

```diff
@@ -75,6 +76,7 @@ list( APPEND test_input
 )
 
 list( APPEND test_output
+  testoutput/amsr2_icec_l2p.nc
```

Comment:
You should have `testoutput/coperl4_icethk.nc `instead of `testoutput/amsr2_icec_l2p.nc`